global_helpers/panther_lookuptable_helpers.py

```diff
@@ -14,3 +18,32 @@ def _lookup(self, match_field: str, *keys) -> list or str:
         if isinstance(match, list):
             return [deep_get(match_value, *keys) if match_value else None for match_value in match]
         return deep_get(match, *keys)
+
+    @property
+    def p_matched(self):
```

Comment:
nit: Unit test?

Reply:
there are a few of them, [here's one](https://github.com/panther-labs/panther-analysis/blob/7eec5df10c687164809d8a1e561c32dd9bf168ad/global_helpers/global_helpers_test.py#L1871)